test/testlib/nuodb_database_utilities.go

```diff
@@ -75,12 +81,12 @@ func StartDatabase(t *testing.T, namespaceName string, adminPod string, options
 	AwaitNrReplicasScheduled(t, namespaceName, smPodName, opt.NrSmPods)
 
 	tePodName := GetPodName(t, namespaceName, tePodNameTemplate)
-	AwaitPodStatus(t, namespaceName, tePodName, corev1.PodReady, corev1.ConditionTrue, 120*time.Second)
+	AwaitPodStatus(t, namespaceName, tePodName, corev1.PodReady, corev1.ConditionTrue, 240*time.Second)
```

Comment:
4 minutes for a TE to start seems excessive. Have you encountered such long times? I have only seen it with admins, when the minikube instance needs to pull a new image.

The SM time below is cumulative. Can it really take 5min + whatever time the TE took? (lets call it 6-10 min)

Reply:
Um - difficult to say with precision.
I was having _lots_ of timeout issues on the minikube environment on my laptop.

I can try shortening the timeouts again.
At least part of the problem was an accumulation of incomplete tests in my minikube environment.

Reply:
I have set the timeouts to 120 secs and 200 secs respectively.
I will see how they perform in Travis.